deel/torchlip/normalizers.py

```diff
@@ -81,7 +82,9 @@ def body_cols(w):
     def body_rows(w):
         return torch.mm(torch.mm(w, w.t()), w)
 
-    def body(w, fct):
+    def body(w, fct, iter):
+        if maxiter - iter > DEFAULT_BJORK_DETACH:
```

Comment:
Not a big deal, but with "strictly greater than", we perform 4 iterations before detach, and not 3 as we could think in `DEFAULT_BJORK_DETACH`.
Maybe we can use `>=` instead?